docs/pages/careers.tsx

```diff
@@ -280,103 +271,40 @@ function CareersContent() {
-            <br /> to build great UIs, faster.
-          </Typography>
-        </Box>
-      </Container>
-      {/* Our ultimate goal */}
+      <Section cozy>
+        <SectionHeadline
+          alwaysCenter
+          overline="Join us"
+          title={
+            <Typography variant="h2" sx={{ maxWidth: 600, mx: 'auto' }}>
+              Build <GradientText>the next generation</GradientText>
+              <br /> of tools for UI development
+            </Typography>
+          }
+          description="Together, we are enabling developers & designers to bring stunning UIs to life with unrivalled speed and ease."
+        />
+      </Section>
       <Divider />
-      <Box
-        sx={(theme) => ({
-          bgcolor: 'grey.50',
-          ...theme.applyDarkStyles({
-            bgcolor: 'primaryDark.900',
-          }),
-        })}
-      >
-        <Section bg="gradient" cozy>
-          <Grid container alignItems="center" spacing={4}>
-            <Grid item xs={12} md={6}>
-              <Typography variant="h2" sx={{ my: 1 }}>
-                Our <GradientText>ultimate</GradientText> goal
-              </Typography>
-              <Typography color="text.secondary" sx={{ mb: 1, maxWidth: 450 }}>
-                We aim high trying to design the most effective and efficient tool for building UIs,
-                for developers and designers. MUI started back in 2014, to unify React and Material
-                Design. Since then, we&apos;ve become a community of over 2M developers from every
-                corner of the world.
-              </Typography>
-              <Typography color="text.secondary" sx={{ mb: 2 }}>
-                We plan on doing all that cultivating our values:
-              </Typography>
-              {[
-                'Customer obsessed. We put our customers front & center.',
-                "Excellence. We're aiming high, and we know it.",
-                'Transparency. Most of our work is public.',
-                'Freedom. We work from anywhere in the world.',
-                'Autonomy. We want to create a safe, high-trust team.',
-              ].map((text) => (
-                <Box key={text} sx={{ display: 'flex', alignItems: 'center', mt: 1 }}>
-                  <IconImage name="pricing/yes" />
-                  <Typography variant="body2" color="text.primary" fontWeight={600} sx={{ ml: 1 }}>
-                    {text}
-                  </Typography>
-                </Box>
-              ))}
-            </Grid>
-            <MuiStatistics />
-          </Grid>
-        </Section>
-      </Box>
+      <OurValues />
       <Divider />
       {/* Perks & benefits */}
-      <Section bg="transparent" cozy>
-        <Grid container alignItems="center" spacing={{ xs: 2, sm: 4 }}>
-          <Grid item xs={12} md={6} sx={{ pr: { sm: 0, md: 4 } }}>
-            <Typography variant="h2" sx={{ my: 1 }} id="perks-amp-benefits">
```

Comment:
I restored the `id` in 396c74673a201e7c3bfe84ee3a23606309be0182, to fix the anchor links.

Reply:
Thank you! My bad on removing it 🤦